certify-postgres-landregistry.properties

```diff
@@ -0,0 +1,139 @@
+# this source code form is subject to the terms of the mozilla public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+## DataProvider plugin specific changes
+
+mosip.certify.issuer=CertifyIssuer
+mosip.certify.vcformat.vc.expiry=true
+# optional field to specify VC signature algo for CertifyIssuer, defaults to Ed25519Signature2020
+# other option: [RsaSignature2018]
+mosip.certify.issuer.vc-sign-algo=Ed25519Signature2020
+mosip.certify.issuer.pub.key=did:web:vharsh.github.io:DID:devcp2#key-0
+mosip.certify.issuer.uri=did:web:vharsh.github.io:DID:devcp2
+mosip.certify.issuer.svg.template.id=5b9c2a12-810a-7388-2dc8-13ee7ad88bac
+mosip.certify.integration.data-provider-plugin=PostgresDataProviderPlugin
+# Mimoto -- INJIMOB-2446
+mosip.certify.issuer.id.field.prefix.url=https://mosip.io/credential/
```

Comment:
can we rename this to mosip.certify.postgres.data-provider-plugin.id-field-prefix-uri

Reply:
This configuration is not specific to the Postgres plugin, this is the workaround we've added so that the Mimoto doesn't set the `id` field to null in case the Credential doesn't have the `id` field.

Reply:
then lets rename this to mosip.certify.data-provider-plugin.id-field-prefix-uri and provide empty value with comments regarding this property.
Also this needs to be moved to the certify-default.properties file. 
Lets keep our usecase default credential templates with id for now so we have avoid this property in the plugin-usecase property files.
I believe this is a temporary workaround, lets we can silently remove this from default property files also once mobile fixes the limitation.

Reply:
Added field in certify-default with empty value and appropriate comment